Add English language Guide for PHP module (#61)
framgia/Training-Guideline: Laravel/laravel.en.md

# Laravel Tutorial Guidline

## 1. Goal
- Get acquainted and use framework Laravel.
- Apply Sun* GIT flow.

## 2. Training Time

> 8 days (Maximine 10 days)

Reviewer: ```suggestion
8 days (Maximum: 10 days)
```

> 8 days (Maximine 10 days)

Reviewer 2: ```suggestion
8 days (Maximize 10 days)
```

> Inlude: 5 days of self-study -> 1st review -> 3 days of doing Quicktask -> 2nd review

Reviewer: ```suggestion
Including: 5 days of self-study -> 1st review -> 3 days of doing Quicktask -> 2nd review
```

> Inlude: 5 days of self-study -> 1st review -> 3 days of doing Quicktask -> 2nd review

Reviewer 2: ```suggestion
Include: 5 days of self-study -> 1st review -> 3 days of doing Quicktask -> 2nd review
```

> Inluding: 5 days of self-study -> 1st review -> 3 days of doing Quicktask -> 2nd review

Reviewer: ```suggestion
Including: 5 days of self-study -> 1st review -> 3 days of doing Quicktask -> 2nd review
```

## 3. Basic knowledge:
1. Init project, Directory Structure
2. Migration
3. Seeder, factory, faker
4. Model : 
	- Mass assignment
	- Relationship 
	- Accessors, mutators
	- Eager loading
	- Scope
5. Route:
	- Route + Resource controller
	- Middleware
	- REST & RESTful
6. Controller:
	- Auth
	- Eloquent orm + query builder
	- CRFS Protection
7. Advance: 
	- Transaction laravel, Helpers, Localization(i18n), repository, cache
	- Blade Template
	- Bower, laravel mix
	- Service provider, service container

## 4. Document Link:
* Document: https://laravel.com/docs
* Book: https://learninglaravel.net/books/laravel
* Ebook: https://goo.gl/hft96o

## 5. Laravel Quicktask Guidline:
- Proceed to do Laravel Quick task according to the following file instructions: **(Skip STEP 3)** https://docs.google.com/document/d/18s2S7JGEvhCUbITNnK5J0veaC5C2Oc1SzKr66Hi0a3o/edit

> - Create pull request and follow https://laravel.com/docs/5.2/quickstart, kết hợp với yêu cầu khác của trainer (trong khi code có thể trình bày code theo cách mình hiểu về flow của Laravel).

Reviewer: phần này vẫn viết tiếng Việt anh ơi.
```suggestion
- Create pull request and follow https://laravel.com/docs/5.2/quickstart, combine with some extra requirement from your trainers (while coding you may write it in the way of how you understand the flow of Laravel).
```

> - Create pull request and follow https://laravel.com/docs/5.2/quickstart, kết hợp với yêu cầu khác của trainer (trong khi code có thể trình bày code theo cách mình hiểu về flow của Laravel).

Reviewer 2: dòng này còn tiếng việt nè anh

- Please read the company's convention file to make the code more correct: https://github.com/framgia/coding-standards/blob/master/vn/README.md#php
- If using sublime text, you can configure the following code to help control the company's convention, by:

> Preference -> Setting User -> clear all and pase this code bellow:

Reviewer 2: ```suggestion
Preference -> Setting User -> clear all and paste this code bellow:
```
